doc/user/content/integrations/connection-pooling.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Connection Pooling"
+description: "How to use connection pooling with Materialize"
+menu:
+  main:
+    parent: "integrations"
+    weight: 10
+    name: "Connection Pooling"
+---
+
+Because Materialize is wire-compatible with PostgreSQL, you can use any
+PostgreSQL connection pooler with Materialize. In this guide, we’ll cover how to
+use connection pooling with Materialize, alongside the common tool PgBouncer.
+
+## PgBouncer
+[PgBouncer](https://www.pgbouncer.org/) is a popular connection pooler for
+PostgreSQL. It provides a lightweight and efficient way to manage database
+connections, reducing the overhead of establishing new connections and
+improving performance.
+
+You can [install PgBouncer](https://www.pgbouncer.org/downloads/) on your local machine
+or on a server. Once installed, you can configure it to connect to your Materialize
+instance. Follow the directions to locate the configuration file for your environment.
+Here’s a basic configuration example:
+
+```ini
+[databases]
+;; Define a database named "materialize" that connects to your Materialize instance
+;; In cloud this is the connection info found in the console
+materialize = host=localhost port=6877
+
+[pgbouncer]
+logfile = /var/log/pgbouncer/pgbouncer.log
+pidfile = /var/run/pgbouncer/pgbouncer.pid
+;; Listen on localhost:6432 for incoming connections
+listen_addr = localhost
+listen_port = 6432
+
+;; Set the authentication type and user list file
+;; Materialize supports either plain or scram-sha-256
+;; For Materialize Cloud, use plain:
+auth_type = plain
+;; For self-hosted Materialize, you can optionally use scram-sha-256:
+;; auth_type = scram-sha-256
+auth_file = /etc/pgbouncer/userlist.txt
```

Comment:
Maybe break it apart and highlight the scram-sha-256 one instead of plain? 

```suggestion
;; Set the authentication type
;; Materialize supports both plain and scram-sha-256.
;; Cloud and Self-Managed support plain.
;; auth_type = plain
;; Self-Managed also supports scram-sha-256:
auth_type = scram-sha-256

;; Set the authentication user list file
auth_file = /etc/pgbouncer/userlist.txt
```